apps/custom-config/base-config.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+var CloudGW = require('@yoda/cloudgw')
+var flora = require('./singleton-flora')()
+/**
+ * base config class
+ */
+class BaseConfig {
+  constructor (activity) {
+    this.activity = activity
+    this.floraAgent = flora
+    this.cloudgw = null
+  }
+
+  /**
+   * url map getter
+   * @returns {null} return null by default
+   */
+  getUrlMap () {
+    return null
+  }
+
+  /**
+   * intent map getter
+   * @returns {null} return null by default
+   */
+  getIntentMap () {
+    return null
+  }
+
+  /**
+   * ready for cloudgw
+   * @param {object} cloudgwConfig
+   */
+  ready (cloudgwConfig) {
+    this.cloudgw = new CloudGW(cloudgwConfig)
```

Comment:
The `ready()` gets called many times, that creates duplicated `CloudGW` objects, that's unnecessary.